### DISCUSSION TITLE
Installing Apple's Swift Programming Language

### DISCUSSION THREAD

**Discussion by goniapt** [oracle/oracle-linux#53] [General Oracle Linux questions]
I am new to Oracle Linux. I have a successful installation of VirtualBox hosting Oracle Linux 8. Now I'm interested in getting the Open Source Swift programming language installed. 

Looking for pointers and resources that would help me in getting started.If there are better forums to use in getting an answer to this question please share. 

Thank you!

**Comment by totalamateurhour** [comment]
Swift is not included in Oracle Linux (8, or otherwise). Your best bet is to try to install bits for CentOS 8 from https://www.swift.org/download/. I'm afraid I don't have any experience with this.

**Comment by Halesharry** [comment]
Oracle Linux excludes Swift from its software (8, or otherwise). Try installing CentOS 8 bits from https://www.swift.org/download/; that's your best choice. I'm sorry, but I've never. <a href="https://financeinvestation.com/">.</a>
